packages/@aws-cdk/aws-appmesh/test/test.virtual-gateway.ts

```diff
@@ -399,6 +600,9 @@ export = {
         backendDefaults: {
           tlsClientPolicy: {
             validation: {
+              subjectAlternativeNames: {
```

Comment:
General comment in this entire PR: can we not modify existing tests, beyond making them compile if there were any breaking changes?

If we want to test something added in this PR, let's create a separate test for it, testing just that.

Reply:
Well noted. Reverting any changes made on existing tests and creating separate test.